models/bert/__main__.py

```diff
@@ -71,8 +71,9 @@ def evaluate_split(model, processor, tokenizer, args, split='dev'):
 
     args.is_hierarchical = False
     processor = dataset_map[args.dataset]()
-    pretrained_vocab_path = PRETRAINED_VOCAB_ARCHIVE_MAP[args.model]
-    tokenizer = BertTokenizer.from_pretrained(pretrained_vocab_path)
+    if not args.hf_checkpoint:
```

Comment:
I am not sure I understand why we need this condition here

Reply:
The flag `--hf-checkpoint` was just allowing to specify one of the hugging face model [checkpoints](https://huggingface.co/transformers/pretrained_models.html) which is automatically loaded making it easier to start training than having to download and move the weights to the `hedwig-data` directory.

However if the weights are included in `hedwig-data` then there's no need for the flag.

Reply:
We didn't want to have the option to download a checkpoint in order to protect against any changes to these checkpoints by hugging-face, which would invalidate our results

Reply:
Oh makes sense.